### DISCUSSION TITLE
Access Policy

### DISCUSSION THREAD

**Discussion by AzgadAGZ** [zscaler/terraform-provider-zia#75] [Q&A]
Took a look on the examples and documentation but cannot find the access policy resource to link segment groups with SCIM groups in ZPA, there is any available or this resource is not working yet?

@willguibr

**Comment by willguibr** [comment]
@AzgadAGZ 
Your questions is related to the ZPA provider, but your comment is being posted under the ZIA provider. Can you please send your question via the proper provider with some more details of what you're trying to achieve?

**ZPA Provider** https://github.com/zscaler/terraform-provider-zpa
Thank you
